src/main/java/com/codessquad/qna/domain/user/HttpSessionUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.codessquad.qna.domain.user;
+
+import com.codessquad.qna.exception.UserNotFoundException;
+
+import javax.servlet.http.HttpSession;
+
+public class HttpSessionUtils {
+    public static String USER_SESSION_KEY = "sessionedUser";
+
+    public static boolean isLoginUser(HttpSession session) {
+        return session.getAttribute(USER_SESSION_KEY) == null ?
+                false : true;
```

Comment:
간단하게 리팩토링이 가능할 것 같아요.

```suggestion
        return session.getAttribute(USER_SESSION_KEY) != null

Reply:
훨씬 깔끔해지네요. 감사합니다👍